src/navigation-bar.tsx

```diff
@@ -0,0 +1,524 @@
+            height: `calc(100vh - ${MOBILE_NAVBAR_HEIGHT}px)`,
+            overflowY: 'auto',
+            background: theme.colors.background,
+            boxShadow: ({menuTransitionState}) =>
+                menuTransitionState !== 'closed' ? `6px 0 4px -4px rgba(0, 0, 0, ${shadowAlpha})` : 'none',
+            transition: `transform ${BURGER_MENU_ANIMATION_DURATION_MS}ms ease-out`,
+        },
+        burgerMenuEnter: {
+            transform: 'translate(-100vw)',
+        },
+        burgerMenuEnterActive: {
+            transform: 'translate(0)',
+        },
+        burgerMenuExit: {
+            transform: 'translate(0)',
+        },
+        burgerMenuExitActive: {
+            transform: 'translate(-100vw)',
+        },
+    };
+});
+
+type MainNavigationBarSection =
+    | {href: string; to?: undefined; onPress?: undefined; title: string}
+    | {to: string; href?: undefined; onPress?: undefined; title: string}
+    | {onPress: () => void; to?: undefined; href?: undefined; title: string};
+
+type MainNavigationBarProps = {
+    sections: Array<MainNavigationBarSection>;
+    selectedIndex?: number;
+    right?: React.ReactElement;
+    logo?: React.ReactElement;
+    isInverse?: boolean;
+    children?: undefined;
+    topFixed?: boolean;
+};
+
+type MenuTransitionState = 'closed' | 'opening' | 'open' | 'closing';
+
+export const MainNavigationBar: React.FC<MainNavigationBarProps> = ({
+    sections,
+    selectedIndex,
+    right,
+    logo,
+    isInverse = false,
+    topFixed = true,
+}) => {
+    const [isMenuOpen, setIsMenuOpen] = React.useState(false);
+    const [menuTransitionState, setMenuTransitionState] = React.useState<MenuTransitionState>('closed');
+    const menuId = useAriaId();
+    const classes = useStyles({isMenuOpen, isInverse, menuTransitionState});
+    const {isTabletOrSmaller} = useScreenSize();
+    const setModalState = useSetModalState();
+
+    logo = logo ?? <NavigationBarLogo />;
+
+    if (isTabletOrSmaller) {
+        const openMenu = () => {
+            setIsMenuOpen(true);
+            setModalState({isModalOpen: true});
+        };
+
+        const closeMenu = () => {
+            setIsMenuOpen(false);
+            setModalState({isModalOpen: false});
+        };
+
+        const disableFocusTrap = menuTransitionState !== 'open';
+
+        return (
+            <>
+                <FocusTrap disabled={disableFocusTrap} group="burger-menu-lock">
+                    <header>
+                        <ThemeVariant isInverse={isInverse}>
+                            <div className={classnames(classes.navbar, {[classes.topFixed]: topFixed})}>
+                                <ResponsiveLayout>
+                                    <Inline space="between" alignItems="center">
+                                        <Inline space={24} alignItems="center">
+                                            <IconButton
+                                                aria-live="polite"
```

Comment:
Is this aria-live working here? is it intended to read the menu options? or the selected option?

Reply:
Yes. It reads the close/open icon button label when it changes